quickwit/quickwit-ingest/src/ingest_v2/workbench.rs

```diff
@@ -138,6 +246,12 @@ impl IngestWorkbench {
             );
             return;
         };
+        if let Some(publish_tracker) = &mut self.publish_tracker {
```

Comment:
I don't understand the mut here.
```suggestion
        if let Some(publish_tracker) = &self.publish_tracker {
```

Reply:
leftover from a previous mutable design. Quite surprised clippy didn't catch this!